docs/system_overview/configuration.md

```diff
@@ -150,7 +150,7 @@ The `local` entry can define multiple addresses for the local server.
 
 The `remote` entries each define a remote server with a set of aliases.
 
-The first address in each stanza is the preferred address and will be used for connecting to clients and remote servers.  The additional addresses are the aliases.
+The first address in each stanza is the preferred address and will be used for connecting to clients and remote servers. This first address should be the hostname of the machine in question and is expected to match the result of running `hostname` on said host. The additional addresses are the aliases.
```

Comment:
two suggestions...

`s/machine/server/`

`s/said host/that server/`

and I think we're good.

Reply:
Done